tiatoolbox/models/architecture/unet.py

```diff
@@ -1,10 +1,10 @@
-"""Defines a set of ResNet variants to be used within tiatoolbox."""
+"""Defines a set of UNet variants to be used within tiatoolbox."""
 
 from typing import List, Tuple
 
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
+import torch.nn.functional as nn_func
```

Comment:
```suggestion
import torch.nn.functional as functional
```

Reply:
Do you want me to fix it in this PR or #345?

Reply:
Let's fix this in this PR. I can close the other PR once we have merged this PR.

BTW, what's the status of this PR? Do you need reviews on this PR still?

Reply:
The PR is ready to be merged really, it has been reviewed several time and applied all the comments. However, I requested for @simongraham  and @vqdang but not received any feedback yet.

Reply:
@simongraham and @vqdang  please can you confirm if you are happy with this PR?